manifests/component_metadata.yaml

```diff
@@ -0,0 +1,6 @@
+releases:
+  - name: ODH Dashboard
+    version: 2.31.0
```

Comment:
i think the intention is dashboard would not need to have a component_metadata.yaml for them, since there is no upstream for dashboard.

Reply:
@andrewballantyne / @lucferbux ^ how you feel adding this for dashboard now?

Reply:
@zdtsw @andrewballantyne Thank you for taking a look at this. 

I can remove the `name`, `version`, and/or `repoUrl` fields if that's preferred but my understanding is the operator needs at least the `supportedArchitectures` field for the Power/Z phase 1 support.

CC: @csams

Reply:
I think @csams this is something you can help out with? I'm not sure what to do on the side of supported architectures...